client/my-sites/stats/site.jsx

```diff
@@ -147,21 +147,20 @@ class StatsSite extends Component {
 		this.props.recordTracksEvent( 'calypso_stats_parsely_banner_click' );
 	};
 
-	buildPrivateSiteBanner() {
+	renderPrivateSiteBanner( slug ) {
```

Comment:
Nit: given that we're using a dedicated argument, it may be worth calling this variable `siteSlug`. (Definitely not blocking!)
```suggestion
	renderPrivateSiteBanner( siteSlug ) {
```